msal/__main__.py

```diff
@@ -204,7 +206,7 @@ def _main():
             header="Input authority (Note that MSA-PT apps would NOT use the /common authority)",
             accept_nonempty_string=True,
             ),
-        allow_broker=allow_broker,
+        enable_broker=enable_broker,
```

Comment:
I thought we decided on an OS explicit flag, i.e. enable on Windows, enable on Mac etc. ?

Reply:
We discussed in person: 

- MSAL should fallback to browser if msal_runtime cannot be initialized (i.e. broker not available)
- an opt-in flag for each operating system

Reply:
> We discussed in person:

Yes. My message above was sent in the last minute before our discussion yesterday.

> * MSAL should fallback to browser if msal_runtime cannot be initialized (i.e. broker not available)

ACK. I double checked the current implementation. It [differentiates two different kinds of errors](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.24.0/msal/broker.py#L23-L27) during the construction. 

1. If app developer opted in but the package PyMsalRuntime was not installed, MSAL Python emits a helpful error message to guide them install the dependency. Error-out is desirable here because it is actionable to the app developers.
2. If app developer opted in and PyMsalRuntime was also installed but failed in its initialization, MSAL Python [emits logs and falls back](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.24.0/msal/application.py#L596-L600).

> * an opt-in flag for each operating system

I believe we meant "proposal 3" above (now I **highlighted** the differences between proposal 2 & 3 in the [original message](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/596#discussion_r1331939378)). Specifically, we are going with "3 flags, one per OS", and MSAL will automatically ignore "enable_broker_on_X" when current platform is Y.

Let me work out a new commit for that.

Reply:
Thank you @rayluo , that makes sense.